cedar-policy-core/src/extensions/decimal.rs

```diff
@@ -104,15 +109,14 @@ impl Decimal {
         // check that the string matches the regex
         // PANIC SAFETY: This regex does parse
         #[allow(clippy::unwrap_used)]
```

Comment:
Probably will need to move this up to the static block.

Reply:
Yeah that's why clippy is angry. Will fix shortly.

Reply:
Should be fixed -- clippy didn't seem to like me annotating a global definition, so I moved the regex into the module with the other constants.